src/mavoscript.js

```diff
@@ -29,7 +29,7 @@ var _ = Mavo.Script = {
 			if (Array.isArray(a)) {
 				result = [
 					...b.map((n, i) => o.scalar(a[i] === undefined? o.identity : a[i], n)),
-					...a.slice(b.length)
+					...a.slice(b.length).map((n) => o.scalar(n, o.identity))
```

Comment:
I wonder if at this point it would be more readable to just loop over both, like so:

```js
var max = Math.max(a.length, b.length);
var ret = [];

for (let i=0; i<max; i++) {
	ret[i] = o.scalar(
		a[i] === undefined? o.identity : a[i],
		b[i] === undefined? o.identity : b[i]
	);
}

return ret;
```

Reply:
Ya, considering my potential future implementations for left and right scalars and identities, I think this would be quicker to pick up.

Reply:
Probably faster too (though I imagine the difference is negligible).

Reply:
Yes. But for my future implementation of right and left scalars, I may need to use the original syntax since I could be using a different scalar for the leftover items of an uneven array.